app-backend/build.sbt

```diff
@@ -55,6 +44,18 @@ scalaVersion in ThisBuild := Version.scala
   * Shared settings across all subprojects
   */
 lazy val sharedSettings = Seq(
+  // We are overriding the default behavior of sbt-git which, by default,
+  // only appends the `-SNAPSHOT` suffix if there are uncommitted
+  // changes in the workspace.
+  version := {
+    // TODO: leave an explaination as to why this is here. e.g. Vagrant -> not
+    // mounting .git -> separate sbt container for development
+    if (git.gitHeadCommit.value.isEmpty) "dev"
```

Comment:
Some of the containers used to compile Scala do not mount the `.git` folder in as a compromise made with Vagrant, so we can't always rely on `git describe` working to version artifacts.